lib/shopify_api/rest/base.rb

```diff
@@ -283,10 +289,16 @@ def to_hash
           end.to_sym
 
           if self.class.has_many?(attribute)
-            hash[attribute.to_s] = get_property(attribute).map(&:to_hash).to_a if get_property(attribute)
+            hash[attribute.to_s] = get_property(attribute).map do |element|
+              data = element
```

Comment:
Nit: is it necessary to rename `element` here?

Reply:
I've refactored this to be a private method, so if you could take another look, that would be great (the Sorbet stuff confused me, so I'm not sure I've done it correctly).